src/lib/solana/index.ts

```diff
@@ -62,6 +123,19 @@ export class SolanaUtils {
     }
   }
 
+  // Phantom or Privy Connected Solana Wallets
+  static async getSolanaProvider(
+    privyConnectedSolanaWallets: PrivyConnectedWalletsParams,
+  ): Promise<SolanaWalletProvider | null> {
+    const provider = await this.getPhantomProvider();
+    if (provider) {
+      return new BrowserWalletProvider(provider);
+    }
+    const { wallets, ready } = privyConnectedSolanaWallets;
+    if (ready && wallets.length > 0) return new PrivyWalletProvider(wallets[0]);
```

Comment:
in the current setup, it looks like a user can only connect 1 wallet with privy. 

Also this should include embedded wallets -> might add this in a separate PR tho. 

RN to keep things simple lets use privy connected sol wallets as a fallback if phantom wallets are not available

Reply:
This will be removed. We will only be using linked privy wallets, or legacy embedded wallets. 

server wallets are still pending access granted in privy. So we should not used them nor should we allow funding to these wallets because once u fund them, you can't withdraw the funds until server wallets are enabled. This is dangerous.